content/v3/pulls/comments.md

```diff
@@ -62,7 +62,7 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
```

Comment:
I dont see how it would be easier to translate from file line number to patch or diff line number.

I dont know how to make a note. You may close PR or merge my change, up to you =)